<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialog_newpbl</class>
 <widget class="QDialog" name="dialog_newpbl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>497</width>
    <height>718</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configure New Problem</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>670</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>451</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Problem Type</string>
   </property>
   <widget class="QRadioButton" name="radBtn_Trad">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>115</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Traditional</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radBtn_Intr">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>115</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Interative</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radBtn_spj">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>141</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Special Judge</string>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="edt_inp">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>451</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_inp">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Input file name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edt_src">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>381</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_src">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Source name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edt_out">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>451</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_out">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Output file name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edt_spj">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>381</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>spj.*</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_spj">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>350</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Special Judge</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edt_score">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>570</y>
     <width>451</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>10</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_score">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>550</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Score per testdata</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edt_prob">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>451</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_prob">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Problem Name</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>451</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Language Support</string>
   </property>
   <widget class="QCheckBox" name="ckBox_Maple">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Maple</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ckBox_MMA">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>141</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Mathematica</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="lbl_src_dir">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>400</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Source Code Directory</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edt_src_dir">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>381</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="btn_src_dir">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>420</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edt_data_dir">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>500</y>
     <width>381</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="btn_data_dir">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>500</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_data_dir">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>480</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Data Directory</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_src_cnt">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>450</y>
     <width>241</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_data_cnt">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>530</y>
     <width>241</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_src_suf">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>270</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_TL">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>600</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Time Limit per test</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edt_TL">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>620</y>
     <width>451</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QToolButton" name="btn_spj">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>370</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>dialog_newpbl</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>dialog_newpbl</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
